profile/models/c5g7/c5g7-cmfd.cpp

```diff
@@ -465,8 +465,8 @@ int main() {
   Cmfd cmfd;
   cmfd.setSORRelaxationFactor(1.5);
   cmfd.setLatticeStructure(51, 51);
-  int cmfd_group_structure[3] = {1,4,8};
-  cmfd.setGroupStructure(cmfd_group_structure, 3);
+  std::vector< std::vector<int> > cmfd_group_structure = {{1,4,7}};
```

Comment:
Since the previous update of the alternative C++ build, the syntax for setting the CMFD group structure has changed. Previously a single array was used to indicate how to collapse the groups (it was a bit complicated, so I'll skip the details). Now, we collapse groups much more straightforward. In particular, if you want to collapse to two groups for CMFD (which is what we want) you would write the following:
```
std::vector< std::vector<int> > cmfd_group_structure = {{1,2,3}, {4,5,6,7}};
```

which would indicate the first three groups would be collapsed into group one and the last four groups would be collapsed into group two.